lib/gtfs/orm/calendar.rb

```diff
@@ -26,7 +26,7 @@ def trips
       end
 
       def calendar_date
-        CalendarDate.where(service_id: service_id)
+        CalendarDate.where(service_id: service_id, date: start_date..end_date)
```

Comment:
please revert this - `service_id` is unique field for Calendar and we always want to have all associated CalendarDate. It's up to the data provider whether to add dates outside the specified range or not